core/src/main/scala/quasar/physical/ts/TSConfig.scala

```diff
@@ -73,15 +75,15 @@ object Auth {
       })
   }
 
-  final case class PrivateKey(value: String) extends Auth
+  final case class PrivateKey(value: String, passphrase: Option[String]) extends Auth
 
-  object PrivateKey extends (String => Auth) {
+  object PrivateKey {
     val Key = "private-key"
   }
 
   final case class Password(value: String) extends Auth
 
-  object Password extends (String => Auth) {
+  object Password {
```

Comment:
Why was this extension removed? It's something the compiler would have inserted *anyway* if we hadn't explicitly defined the companion object, and it allows us to use this in a point-free fashion. Case class companion objects should basically always have this unless they're polymorphic.